allennlp/data/vocabulary.py

```diff
@@ -627,6 +630,29 @@ def add_token_to_namespace(self, token: str, namespace: str = "tokens") -> int:
         else:
             return self._token_to_index[namespace][token]
 
+    def extend_from_precomputed_encoding(
```

Comment:
Can you rename this to `extend_from_dictionary`, and put it up by the other `extend*` methods?